test/index.js

```diff
@@ -62,3 +74,50 @@ describe('rules documentation', () => {
         });
     });
 });
+
+// Tests for SSR rules
+describe('SSR rules exports', () => {
+    SSR_RULE_FILES.forEach((ruleFile) => {
+        const ruleName = path.basename(ruleFile, '.js');
+        const ssrRulePath = `ssr/${ruleName}`;
+
+        it(`should export SSR rule "${ruleFile}"`, () => {
+            assert.equal(
+                plugin.rules[ssrRulePath],
+                require(path.join(SSR_RULES_FOLDER, ruleFile)),
+                `SSR Rule "${ruleName}" is not exported.`,
+            );
+        });
+    });
+});
+
+describe('SSR rules documentation', () => {
```

Comment:
Add a test that the rules are named correctly in the documentation? 😅 Something like "first line starting with `# ` in `<rule>`.md contains `<rule>`".

Reply:
We cant add a test for this as not all rules readme file follow the same pattern of having the rule name in title.Also its very unlikely to happen as here we copied earlier documentation and missed updating those lines.